Eastern-Research-Group/mywaterway
# Bugfix/724 chart tooltip drift

## Related Issues:
* [HMW-724](https://jira.epa.gov/browse/HMW-724)

## Main Changes:
* Added a CSS rule for Highcharts graps to override a transition that is being set in `epa.css`. The transition was causing the CyAN charts tooltip text to drift from its container.
  * This change is in line 350 of `index.html`. All other changes in that file are due to running Prettier.
* Updated the `highcharts` package to fix a bug where "zones" were being dropped from the CyAN histogram, described in [this issue](https://github.com/highcharts/highcharts/issues/20613).
* Removed extra Y padding from two text boxes in the CyAN accordion content.
* Switched the axis title font for the Highcharts column charts and the Visx scatter/line graph (`Source Sans Pro Web` was rendering poorly).

## Steps To Test:
1. Navigate to the CyAN content of http://localhost:3000/community/lake%20mattamuskeet/monitoring. 
2. Verify the chart tooltips follow the mouse without drift, and the axis titles look appropriate.
3. Go to http://localhost:3000/monitoring-report/STORET/21NC03WQ/21NC03WQ-0208458892/ and repeat step 2.
4. Go to http://localhost:3000/state/FL/water-quality-overview, select the **Drinking Water** topic, and make sure the charts look as expected.
